app/main.py

```
@@ -119,6 +120,18 @@ async def get_forecast_summaries(model: ModelEnum, request: schemas.StationCodeL
         LOGGER.critical(exception, exc_info=True)
         raise
 
+@app.post('/noon_forecasts/', response_model=schemas.NoonForecastResponse)
+def get_noon_forecasts(request: schemas.StationCodeList, _: bool = Depends(authenticate)):
+    """ Returns noon forecasts pulled from BC FireWeather Phase 1 website for the specified
+    set of weather stations. """
+    try:
+        start_date = datetime.datetime.now().replace(hour=0, minute=0, second=0)
+        end_date = start_date + datetime.timedelta(days=5)
+        LOGGER.info('Querying /noon_forecasts/ for %s from %s to %s', request.stations, start_date, end_date)
+        return fetch_noon_forecasts(request.stations, start_date, end_date)
+    except Exception as exception:
+        LOGGER.critical(exception, exc_info=True)
```

We really need to add a catch-all exception in - I'll make a ticket for it....